batch/python/raster_tile_cache.py

```diff
@@ -89,7 +89,7 @@ def get_input_tiles(prefix: str) -> List[Tuple[str, str]]:
     return tiles
 
 
-def create_tiles(args: Tuple[Tuple[str, str], str, str, str, str, int, bool, int]):
+def create_tiles(args: Tuple[Tuple[str, str], str, str, str, str, int, bool, int, int]):
```

Comment:
I guess this was already like this, but why is the type here this ridiculously long tuple type lol? Maybe we should consider using named tuples, or defining this as an object.

Reply:
Ugh, the argument is required to be a tuple by the process execution method we're using. Will try making a NamedTuple.